### DISCUSSION TITLE
simvar carburetor

### DISCUSSION THREAD

**Discussion by Gert57** [nguyenquyhy/Flight-Tracker-StreamDeck#363] [Ideas]
Hi Hy,
Is is possible to add the following simvar:
RECIP CARBURETOR TEMPERATURE:index
Now the DHC-2 Beaver is in MSFS, you need this value for leaning and engine monitoring.
Would be very much appreciated if it is possible.

Thanx!

**Comment by nguyenquyhy** [comment]
This is added from v0.47.
